src/api.ts

```diff
@@ -63,10 +63,16 @@ export function makeConsentManagerAPI({
       Promise.resolve(
         handleSetViewState(options?.viewState || 'OptOutDisclosure', auth),
       ),
-    showConsentManager: (options) =>
-      Promise.resolve(
-        handleSetViewState(options?.viewState || 'open', undefined, true),
-      ),
+    // eslint-disable-next-line require-await
+    showConsentManager: async (options) => {
+      if (options?.viewState !== ViewState.TCF_EU) {
+        handleSetViewState(options?.viewState || 'open', undefined, true);
+        return;
+      }
+      logger.warn(
+        'TCF_EU view state is not valid. Please configure your regime to use this view state.',
```

Comment:
```suggestion
        'TCF_EU view state is not valid for this user experience. Please configure your Regional Experience to use this view state and try again.',
```

Reply:
I think we should also only throw this error for TCF_EU explicitly, and if the user throws and other error, we should log out the list of valid view states for them

Reply:
Just to confirm, if a user uses 'banana' as a view state, that's when we list out the valid view state?

Reply:
@linh-transcend yea, that's when i think we say "'${input}' is not a valid view state. Valid view states are: ..."

Reply:
thanks, updated!
![Screenshot 2024-02-08 at 10 22 28 AM](https://github.com/transcend-io/consent-manager-ui/assets/94718676/44d862af-b9cb-4eb5-a4e2-52fcd2adf563)